### DISCUSSION TITLE
how can I make the level-1 menu (doc name) display content, instead of directing another page?

### DISCUSSION THREAD

**Discussion by xuejiao-joy** [facebook/docusaurus#8984] [Q&A]
In the following example, after I click "Data Type Reference" in the sidebar, the content displayed is redirecting to "my-home-doc" page, like this:
![image](https://github.com/facebook/docusaurus/assets/107540910/8c4e1fe5-c1bc-4833-8b4a-009542d78a3d)
actually, I am expecting that the content displayed is specifically written for this reference, for example, an overview about this reference, like this:
![image](https://github.com/facebook/docusaurus/assets/107540910/6fb6a441-8b08-4aac-9a69-d1cb318ea4a6)
In this example, after I click "Guides", the "Docsaurus Guides" page which is not displayed in the sidebar is displayed. 

How could I achieve this?

**Comment by pranabdas** [comment]
I think you need to set category `link` type as `generated-index` in `sidebars.js`:

https://github.com/facebook/docusaurus/blob/9af32ae0d540a802be8807d8498c808e569c820e/website/sidebars.js#L17-L19

**Reply by xuejiao-joy** [reply]
I tried but an error was reported...
my `sidebars.js` looks largely different from yours... mine likes this: 
```
/**
 * Creating a sidebar enables you to:
 - create an ordered group of docs
 - render a sidebar for each doc of that group
 - provide next/previous navigation

 The sidebars can be generated from the filesystem, or explicitly defined here.

 Create as many sidebars as you want.
 */

// @ts-check

/** @type {import('@docusaurus/plugin-content-docs').SidebarsConfig} */
const sidebars = {
  // By default, Docusaurus generates a sidebar from the docs folder structure
  docSidebar: [{type: 'autogenerated', dirName: '.'}],

  // The following style split the docs to separate parts of docs
  // gettingStartedSidebar: [{type: 'autogenerated', dirName: 'Getting Started'}],
  // guideSidebar: [{type: 'autogenerated', dirName: 'Guide'}],
  // referenceSidebar: [{type: 'autogenerated', dirName: 'Reference'}],

  // But you can create a sidebar manually
  /*
  tutorialSidebar: [
    'intro',
    'hello',
    {
      type: 'category',
      label: 'Tutorial',
      items: ['tutorial-basics/create-a-document'],
    },
  ],
   */
};

module.exports = sidebars;
```
Do I need to rewrite the whole `sidebars.js` file like yours?

**Comment by pranabdas** [comment]
It should work:
```
...
tutorialSidebar: [
    'intro',
    'hello',
    {
      type: 'category',
      label: 'Tutorial',
      link: { 
          type: 'generated-index', 
      },
      items: ['tutorial-basics/create-a-document'],
    },
  ],
...
```
You should not set the `slug` of any of the pages under the category to the category link (then the URLs would collide). What the error message say? Or do you have a demo/public repository reproducing the problem that we can look into?
